automation/scripts/create_migration_mysql.sh

```diff
@@ -57,6 +57,6 @@ done
 
 export PYTHONPATH=$ROOT_DIR
 
-alembic -c "${ROOT_DIR}/server/api/alembic_mysql.ini" upgrade head
-alembic -c "${ROOT_DIR}/server/api/alembic_mysql.ini" revision --autogenerate -m "${MLRUN_MIGRATION_MESSAGE}"
-
+cd ${ROOT_DIR}/server/api
```

Comment:
IIRC if you cd you change the cwd which may cause errors when running this as part of some make command